src/python/platforms/android/adb.py

```diff
@@ -320,6 +320,10 @@ def is_gce():
   android_serial = environment.get_value('ANDROID_SERIAL')
   return android_serial.startswith('127.0.0.1:')
 
+def is_automotive():
+  """Returns if we are running in Android Automotive OS, currently only for Osprey."""
+  model = get_property('ro.product.model')
+  return model.startswith('Osprey')
```

Comment:
Why startswith, why not == ?, do we have model examples of like Osprey_ ?

Reply:
I agree to use ==, because there is not variations for now